docs/cluster_spec.md

```diff
@@ -1,10 +1,9 @@
 # Description of Keys in `config` and `cluster.spec`
 
-This list is not complete, but aims to document any keys that are less than self-explanatory.
+This list is not complete but aims to document any keys that are less than self-explanatory. Our [godoc](https://godoc.org/k8s.io/kops/pkg/apis/kops) reference provides a more detailed list of API values. [ClusterSpec](https://godoc.org/k8s.io/kops/pkg/apis/kops#ClusterSpec), defined as `kind: Cluster` in YAML, and [InstanceGroup](https://godoc.org/k8s.io/kops/pkg/apis/kops#InstanceGroup), defined as `kind: InstanceGroup` in YAML, are the two top-level API values used to describe a cluster.
```

Comment:
Yah I can add some links to the sections. I want to get the same doc gen that they have for kubernetes.io running on our apis. Not sure how. It is an open issue